.gitignore

```diff
@@ -2,4 +2,6 @@ build/
 logs/
 nbproject/
 vendor/
+.idea/
```

Comment:
Well, I work with many OS and VMs, often having the same php app mounted under both linux and windows.
Thus I do not have always the same global environment configuration available